packs/src/classes/paladin.json

```diff
@@ -88,7 +191,8 @@
         "value": {},
         "level": 3,
         "title": "Features",
-        "icon": "systems/dnd5e/icons/svg/item-grant.svg"
+        "icon": "systems/dnd5e/icons/svg/item-grant.svg",
+        "_id": "ki8eEksoSjfOHDcF"
```

Comment:
Should we add a trait advancement to apply immunity to disease from the Divine Health feature?